packages/typespec-test/test/openai_modular/spec/models/images.tsp

```diff
@@ -0,0 +1,143 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/http";
+import "@typespec/versioning";
+
+using Azure.Core;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+namespace Azure.OpenAI;
+
+@doc("The desired size of the generated images. Must be one of 256x256, 512x512, or 1024x1024.")
+@added(ServiceApiVersions.v2023_06_01_Preview)
```

Comment:
They are already using multi-api now ?

Reply:
Not sure the detailed background, the relevant pr is here: https://github.com/Azure/azure-rest-api-specs/pull/24382